src/main/java/cascading/tap/hive/HivePartitionTap.java

```diff
@@ -123,6 +160,91 @@ public void sourceConfInit( FlowProcess<? extends Configuration> flowProcess, Co
     super.sourceConfInit( flowProcess, conf );
     }
 
+
+  @Override
+  public String[] getChildPartitionIdentifiers( FlowProcess<? extends Configuration> flowProcess, boolean fullyQualified ) throws IOException
+    {
+    Configuration conf = flowProcess.getConfig();
+    IMetaStoreClient metaStoreClient = null;
+    try
+      {
```

Comment:
I've only tested it locally so far, so 1.0.0. I'm trying to put together a Demo class in the demo project.

I've dropped the `hiveVersion` in build.gradle to 0.12.0 and the usage of `IMetaStoreClient` is still valid.

However, when I drop to 0.14.0 another area of `HivePartitionTap` breaks. 
https://github.com/Cascading/cascading-hive/blob/2.0/src/main/java/cascading/tap/hive/HivePartitionTap.java#L101 
`AcidUtils.baseFileFilter` is only available from 1.0.0. The impl of this could easily be copied over.

Also, if I drop to 0.13.x then more stuff breaks in one of the tests.